kalliope/signals/order/order.py

```diff
@@ -83,6 +83,9 @@ def __init__(self):
         self.machine.on_enter_start_order_listener('start_order_listener_thread')
         self.machine.on_enter_waiting_for_order_listener_callback('waiting_for_order_listener_callback_thread')
         self.machine.on_enter_analysing_order('analysing_order_thread')
+
+	# add thread to interrupt when sleeping
+        self.Wakeup = Event()
```

Comment:
why using an "Event" object here?

Reply:
Wakeup should be lower case to respect python PEP

Reply:
This is used to catch the SIGCONT signal and wake up the trigger.
def wakeup has the code that calls the Event().set function which
release the thread and activates the trigger.

Why there? It seemed the right place to me? I have little to no python experience or knowledge.

Thank you,  I will change Wakeup to lower case

Reply:
I mean, why using Event? And not a Thread?

Reply:
From reading the docs Event() appeared to be the proper object and when I tried it, it worked. I repeat, I know little to no python.

Reply:
Event is a Kalliope signal. Not designed to be used here. 
And following what I can see here you use only the Thread object behavior.

Reply:
I do not believe so. From the top of order.py
from threading import Thread, Event

Reply:
There is also different sizes of indentations, sometimes you use 3 spaces, sometimes 2. You should use the same indentation as the rest of the code, its 4 spaces.

Reply:
Yes I can see that you've imported Event. What I mean is that you use only the "Thread' behavior of the Event object as [it's a parent object](https://github.com/kalliope-project/kalliope/blob/master/kalliope/signals/event/event.py#L12).
You don't need to instantiate an Event object here but a simple native thread.